packages/@dcl/ecs/src/components/extended/Animator.ts

```diff
@@ -0,0 +1,94 @@
+import { ComponentDefinition, Entity, PreEngine } from '../../engine'
+import { PBAnimationState } from '../generated/pb/decentraland/sdk/components/animator.gen'
+import * as AnimatorSchema from './../generated/Animator.gen'
+
+/**
+ * @public
+ */
+export interface AnimatorComponentDefinition extends ComponentDefinition {
+  /**
+   * @public
+   *
+   * [Helper] Get a `mutable` version of animator clip
+   * @param entity
+   * @param name
+   */
+  getClip(entity: Entity, name: string): PBAnimationState | null
```

Comment:
Maybe we need to follow our pattern of getSomethingOrNull ?  `getClipOrNull`.
Not sure, im not a big fan of Component.getOrNull. But thats another discussion

Reply:
I think the `orNull` should be deprecated. It always exists the possibility that something is undefined, so if you are really sure that it doesn't, you can use `!`, and the animation can be a good use-case

Reply:
The convention so far was that we never use `!` due to linter and error prone of the code, and then orNull can return something or null (this is very functional and monadic)

And if something was `getSomething(x): Something`, it can fail if Something doesn't exist. On the contrary, `getSomethingOrNull(x): Something | null` enforces good type checking and doesn't throw

Reply:
Thanks, that point had gone from me. In that case, I think we can write both functions `getClip` and `getClipOrNull`. 
What about the `play` and `stop` functions? They return `true | false`, true if they find the clip and change the info, false otherwise.

Reply:
sounds reasonable to return a boolean and defaulting to a value in that case